packages/front-end/components/Settings/EditDataSourceSettingsForm.tsx

```diff
@@ -269,6 +280,20 @@ const EditDataSourceSettingsForm: FC<{
                               )
                             }
                           />
+                          <div>
+                            <label style={{ padding: 3 }}>
+                              Add Experiment & Variation Name
```

Comment:
I would change this label to `Use Name Columns` and add a tooltip with more detailed info.  Something like this:

> Enable this if you store experiment/variation names as well as ids in your table

Reply:
Resolved by new changes